addons/web/controllers/main.py

```diff
@@ -477,8 +477,10 @@ def _compute_xml_tree(self, addon, file_name, source):
 
                 xpaths = list(template_tree)
                 if self.debug and inherit_mode == self.EXTENSION_MODE:
-                    for xpath in xpaths:
-                        xpath.insert(0, etree.Comment(" Modified by %s from %s " % (template_name, addon)))
+                    # Comments are a bit messed up in owl 1. So for now, we disable this debug comment.
+                    # for xpath in xpaths:
+                        # xpath.insert(0, etree.Comment(" Modified by %s from %s " % (template_name, addon)))
+                    pass
```

Comment:
```suggestion
                # owl chokes on comments, disable debug comments for now
                if False: # self.debug and inherit_mode == self.EXTENSION_MODE:
                    for xpath in xpaths:
                        xpath.insert(0, etree.Comment(" Modified by %s from %s " % (template_name, addon)))
```
`if False:` compiles to a no-op and the condition we're dropping has no overlap with the `else`.

Reply:
Good to know thanks